packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -699,9 +699,19 @@ export const handlers = [
         type: 'g5-standard-20-s1',
       }),
       linodeFactory.build({
+        label: 'linode_with_tags_test1_test2',
         // eslint-disable-next-line sonarjs/no-duplicate-string
```

Comment:
why this eslint-disabled comment?

Reply:
this wants me to declare new vars all the time, so ignoring